### PR TITLE
Jetpack Cloud: Add settings page

### PR DESCRIPTION
**Pull request by rcoll**
**_Please remember, this is a minimal changeset that brings in the foundation of this page. It merely  sets the stage for a MVP, and is not yet meant to be an MVP. Additions, fixes, can/should be added as future PRs._**

#### Changes proposed in this Pull Request

* Modifies the `RewindCredentialsForm` component to accept a `notices` boolean, allows us to disable the legal text notices
* Adds a connection status module to the top of the settings page (let's make this it's own component at a later date, perhaps for use on the dashboard as well)
* Adds a contextualized `RewindCredentialsForm` to the settings page
* Adds basic styling for large screens throughout (mobile styles need discussion)

#### Testing instructions

* Test the settings page for a site with valid credentials.
* Test the settings page for a site without credentials. Test adding working credentials.
* Ensure no regression in normal Calypso when using the `RewindCredentialsForm` component in the settings section. Ensure that the text notices are still showing.

### CODE REVIEW

File: client/landing/jetpack-cloud/sections/settings/main.jsx [hunk 1]
```diff
@@ -3,22 +3,78 @@
...
+ * Style dependencies
+ */
+import './style.scss';
 
 class SettingsPage extends Component {
 	render() {
-		return <div>Welcome to the settings page for { this.props.siteId }</div>;
+		const { rewind, siteId, translate } = this.props;
+
+		const connected = 'active' === rewind.state;
```
Comment (enejb) [inline client/landing/jetpack-cloud/sections/settings/main.jsx]: Can we update this to `isConnected` ?

File: client/components/rewind-credentials-form/index.jsx [hunk 2]
```diff
@@ -43,11 +43,13 @@ export class RewindCredentialsForm extends Component {
 		siteUrl: PropTypes.string,
 		labels: PropTypes.object,
 		requirePath: PropTypes.bool,
+		notices: PropTypes.bool,
```
Comment (enejb) [inline client/components/rewind-credentials-form/index.jsx]: instead of notices can we update it to `hasNotices` ? this way the prop will be more clear.

Comment (rcoll) [reply client/components/rewind-credentials-form/index.jsx]: done in https://github.com/Automattic/wp-calypso/pull/40094/commits/7f711af2e0c7a17edd40ceed07639db6fd914bb8

File: client/landing/jetpack-cloud/sections/settings/main.jsx (hunk 1 above)
Comment (rcoll) [reply client/landing/jetpack-cloud/sections/settings/main.jsx]: done in https://github.com/Automattic/wp-calypso/pull/40094/commits/7f711af2e0c7a17edd40ceed07639db6fd914bb8

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/3012952

Hi @rcoll, could you please edit the description of this PR and add a screenshot for our translators? Ideally it'd include all of the following strings:
- `Server connection details`
- `Server status: Connected`
- `One-click restores are enabled.`
- `Server status: Not connected`
- `Enter your server credentials to enable one-click restores for Backups. {{a}}Need help? Find your server credentials{{/a}}`

Thank you in advance!

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/3012952).

**Pull request merged**